### PR TITLE
Recommended songs

### LINKED ISSUE #5: Add API Endpoint to recommend youtube playable links in JSON format
This should be done using youtube-dl and pafy

### PR DESCRIPTION
**Pull request by abhiroopdutta**
## Description

Added a function to recommend songs and give out the response in JSON along with song status "song started".  Currently, recommending only 20 songs.

## Motivation and Context

Fixes #5 

## Screenshots (if appropriate):
screenshot from 2018-12-20 02-13-47

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: server/youtube_videos.py [hunk 1]
```diff
@@ -24,6 +24,35 @@ def youtube_search(q, max_results=50,order="relevance", token=None, location=Non
```
Comment (stealthanthrax) [inline server/youtube_videos.py]: Please convert this file in a class and follow OOP principles in the other files as well

File: server/server.py [hunk 2]
```diff
@@ -34,23 +24,35 @@ def youtube():
     Media.get_mrl()
     player.set_media(Media)
     player.play()
-    display_message = {"song":"started"}
-    resp = jsonify(display_message)
+
```
Comment (stealthanthrax) [inline server/server.py]: Create a class for all the non-api functions

File: server/server.py [hunk 3]
```diff
@@ -5,27 +5,17 @@
...
-@app.route('/song', methods=['GET'])
-def youtube():
-    vid = request.args.get('vid')
-
-    
-    search = youtube_search(vid)
-    videoId = search[1][1]['id']['videoId']
-
-    url = 'https://www.youtube.com/watch?v=' + videoId
+#this function takes the song_id and starts playing the song in vlc
```
Comment (stealthanthrax) [inline server/server.py]: Add the documentation of the classes that you will be making

**Comment by abhiroopdutta** [comment]
I have made the changes, please check it out.

**Pull request merged**